pkg/scheduling/volumeusage.go

```diff
@@ -144,7 +145,7 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, pod *v1.Pod, v
 		//  2. The StorageClass never existed and was used to bind the PVC to an existing PV, but that PV was removed
 		// In either of these cases, we should ignore the PVC while computing limits and continue.
 		if errors.IsNotFound(err) {
-			logging.FromContext(ctx).With("pod", pod.Name, "volume", volumeName, "pvc", pvc.Name, "storageclass", storageClassName).Debugf("failed tracking CSI volume limits for volume with unbound PVC, %w", err)
+			log.FromContext(ctx).WithValues("pod", pod.Name, "volume", volumeName, "pvc", pvc.Name, "storageclass", storageClassName).Error(err, "failed tracking CSI volume limits for volume with unbound PVC, %w")
```

Comment:
This is a change in the error severity. Can we leave the severities alone for this PR while we are making so many changes and leave re-classification of severities to a different PR?